app/views/users/index.html.erb

```diff
@@ -6,7 +6,20 @@
         <span class="profile-link">
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
         </span>
+
+        <% if current_user.friend_requests(user)%>
+         <button> pending request</button>
+        <% elsif current_user.receive_invitation(user)%>
+          <%= link_to 'Accept', accept_friend_request_path(user_id: user.id) %>
+          <%= link_to 'Decline', deny_friend_request_path(user_id: user.id), method: :delete%>
+        <% elsif !current_user.friend?(user)  && user.id != current_user.id%>
+          <%= link_to 'Add Friend', send_friend_request_path(user_id: user.id) %>
+        <% elsif current_user.friend?(user)%>
+         <%= link_to 'Unfriend', unfriend_path(user_id: user.id), method: :delete, data: {confirm: "Are you sure you want to unfriend this user?"}  %>
+        <% end %>
```

Comment:
- [ ] put the if/else statement in a helper method that renders the links and call the helper method here instead of having this complex statement directly in the view